rest-api-spec/README.markdown

```diff
@@ -67,6 +67,23 @@ If an API is stable but it response should be treated as an arbitrary map of key
 }
 ```
 
+## Type definition
+In the documentation, you will find the `type` field, which documents which type every parameter will accept.
+
+#### Querystring parameters
+| Type  | Description  |
+|---|---|
+| `list`  | An array of strings *(represented as a comma separated list in the querystring)* |
+| `date` | A string value representing a date *(used only in ML)*   |
```

Comment:
The date may be supplied as a number representing milliseconds since the epoch (1/1/1970) or as an ISO8601 formatted date.

More precisely still, what is supplied must be parseable by the Elasticsearch date format `strict_date_optional_time||epoch_millis` as defined in https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html#built-in-date-formats

Reply:
Thanks! I've updated the document.